seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-plugins/checkpoint-storage-hdfs/pom.xml

```diff
@@ -36,6 +36,12 @@
             <version>3.0.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aliyun</artifactId>
+            <version>3.0.0</version>
```

Comment:
```suggestion
            <version>${hadoop-aliyun.version}</version>
```

Reply:
Used Hadoop3 unified version to all hadoop component version which checkpoint will be dependent on.